### DISCUSSION TITLE
Release Notes - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#982] [Announcements]
# Release Notes - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/release-notes/

**Comment by timorsea** [comment]
新增 火种节点需要永久存在，用户安装 minio、chart museum、registry。
部署完成后火种节点可以关机吗，删除有何影响？

**Reply by jzhupup** [reply]
火种节点目前需要持久化存在，不可删除关机，
火种节点部署了registry、chart museum，5.0 所有的组件的涉及所有镜像都存在火中节点的 registry 中，而且未来新版本内置的模版仓库中 中所有的chart 离线包都放在了 chart museum 中。
